parameters/src/canary/mod.rs

```diff
@@ -0,0 +1,138 @@
+            $crate::insert_canary_key!($map, string, $type<$network>, ("unbond_public", $crate::canary::[<UnbondPublic $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("unbond_delegator_as_validator", $crate::canary::[<UnbondDelegatorAsValidator $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("claim_unbond_public", $crate::canary::[<ClaimUnbondPublic $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("set_validator_state", $crate::canary::[<SetValidatorState $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("transfer_private", $crate::canary::[<TransferPrivate $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("transfer_public", $crate::canary::[<TransferPublic $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("transfer_public_as_signer", $crate::canary::[<TransferPublicAsSigner $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("transfer_private_to_public", $crate::canary::[<TransferPrivateToPublic $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("transfer_public_to_private", $crate::canary::[<TransferPublicToPrivate $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("join", $crate::canary::[<Join $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("split", $crate::canary::[<Split $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("fee_private", $crate::canary::[<FeePrivate $variant>]::load_bytes()));
+            $crate::insert_canary_key!($map, string, $type<$network>, ("fee_public", $crate::canary::[<FeePublic $variant>]::load_bytes()));
+        }
+    }};
+}
+
+#[macro_export]
+macro_rules! insert_canary_key {
+    ($map:ident, $string:tt, $type:ident<$network:ident>, ($name:tt, $circuit_key:expr)) => {{
+        // Load the circuit key bytes.
+        let key_bytes: Vec<u8> = $circuit_key.expect(&format!("Failed to load {} bytes", $string));
+        // Recover the circuit key.
+        let key = $type::<$network>::from_bytes_le(&key_bytes[1..]).expect(&format!("Failed to recover {}", $string));
+        // Insert the circuit key.
+        $map.insert($name.to_string(), std::sync::Arc::new(key));
+    }};
+}
+
+// Inclusion
+impl_remote!(InclusionProver, REMOTE_URL, "resources/", "inclusion", "prover");
+impl_local!(InclusionVerifier, "resources/", "inclusion", "verifier");
+
+/// The function name for the inclusion circuit.
+pub const NETWORK_INCLUSION_FUNCTION_NAME: &str = "inclusion";
```

Comment:
The trait should reference this.

Reply:
The PR was already approved, and this is the same methodology used in the testnet trait